client/app/bundles/course/survey/store.js

```diff
@@ -5,7 +5,7 @@ import rootReducer from './reducers';
 export default ({ surveys }) => {
   const initialStates = surveys;
   const storeCreator = (process.env.NODE_ENV === 'development') ?
-    compose(applyMiddleware(thunkMiddleware, require('redux-logger').logger))(createStore) : // eslint-disable-line global-require
+    compose(applyMiddleware(thunkMiddleware, require('redux-logger').logger))(createStore) : // eslint-disable-line global-require, max-len
```

Comment:
what do you mean by that? this line triggered the max-len warning because of the disable line comment. i added `max-len` as well to stop the warning for this line only